public/style.css

```diff
@@ -553,7 +529,42 @@ input[type=checkbox]:checked + label ~ .talent-list {
   }
 }
 
-@media only screen and (width: 1024px) {
+@media only screen and (min-device-width: 768px) and (max-device-width: 1023px) {
+  html {
+    font-size: 33.33%;
+  }
+
+  .talent-nav {
+    left: 25rem;
+  }
+
+  .about-container {
+    top: -23rem;
+  }
+
+  .projects-title {
+    top: 18rem;
+  }
+
+  .teal-triangle1 {
+    width: 50rem;
+    height: 30rem;
+  }
+
+  .teal-triangle2 {
+    left: 35rem;
+  }
+
+  .more-learners-button {
+    width: 20rem;
+    height: 6rem;
+    top: 2rem;
+    z-index: 1;
+    left: 2rem;
+  }
+}
+
+@media only screen and (min-device-width: 1024px) {
```

Comment:
thank you for doing this CSS refactor! it will come in handy and make future edits way easier =)